src/server/services/transfers/Allocator.cpp

```diff
@@ -38,7 +38,7 @@ Allocator::AllocatorAlgorithm getAllocatorAlgorithm() {
         return Allocator::AllocatorAlgorithm::GREEDY;
     }
     else {
-        return Allocator::AllocatorAlgorithm::INVALID_ALLOCATOR_ALGORITHM;
+        return Allocator::AllocatorAlgorithm::MAXIMUM_FLOW;
```

Comment:
Should default be max flow or greedy? I guess max flow makes sense if we want it to get used in production?